.codespell.exclude

```diff
@@ -0,0 +1,8 @@
+      scavA(_([["Meee? Drinkinging tooo mich? Thatss sshtoopid."
```

Comment:
Would it be possible to annotate this in the file itself instead of storing all the strings we want to ignore in a monolithic file? Would probably make it easier to follow and avoid potential merge conflicts.

Reply:
I don't think it's possible to do that, but I could add comments saying where each line came from.
But yes, merge conflicts can definitely be an issue.

Reply:
It seems possible with https://github.com/codespell-project/codespell/pull/2400 to the best of my understanding. Am I correct? It would also allow implementing on a per line / word or just per line basis.